tools/health/isAlive

```diff
@@ -85,13 +85,6 @@ def isElkAlive() :
     okString = '"cluster_name"'
     return monitorUtil.outputChecker(cmd, okString, 1)
 
-def isKafkaAlive() :
```

Comment:
For the record: Consul didn't even use `isAlive` before.

Still a bit undecided on how to get consul to properly check for Kafka's health, but I think that's a separate issue even? Maybe `controller` and `invoker` expose their Kafka connectivity as part of their `/ping`?

What I'm saying is: This is probably no worse than before in terms of visibility of Kafka's health.